client/containers/Pub/PubDocument/Review/reviewEditor.scss

```diff
@@ -0,0 +1,6 @@
+.review-editor {
```

Comment:
Re: the wrapper div from earlier — it's our convention to call the outermost class in a React component `.my-thing-component` instead of just `.my-thing`.